src/main/java/org/motechproject/newebodac/web/FieldConfigController.java

```diff
@@ -33,4 +34,19 @@
   public FieldConfigDto create(@RequestBody @Valid FieldConfigDto fieldConfigDto) {
     return fieldConfigService.create(fieldConfigDto);
   }
+
+  @RequestMapping(value = "/fieldConfig/update/{id}", method = RequestMethod.PUT)
```

Comment:
I don't think we need the 'create', 'update', 'delete' in the the url, we will know which is which by the method and the parameters